terminal/termbox/cell_options.go

```diff
@@ -28,7 +28,16 @@ func cellColor(c cell.Color) tbx.Attribute {
 
 // cellOptsToFg converts the cell options to the termbox foreground attribute.
 func cellOptsToFg(opts *cell.Options) tbx.Attribute {
-	return cellColor(opts.FgColor)
+	a := cellColor(opts.FgColor)
+	if opts.Bold {
+		a |= tbx.AttrBold
+	}
+	// FIXME: Termbox doesn't have an italics attribute
```

Comment:
How would you feel about returning an error here when the user tried to set italics or strikethrough on termbox? Giving them complete information in an error message might be more user friendly than silently ignoring the requested options.

Reply:
Yeah, that sounds a lot better.